src/python/pants/engine/native.py

```diff
@@ -855,6 +855,12 @@ def write_log(self, msg, level, target):
     target = target.encode()
     return self.lib.write_log(msg, level, target)
 
+  def write_stdout(self, session, msg: str):
+    return self.lib.write_stdout(session, msg.encode())
```

Comment:
Hmm, do these need to be strings or bytes? Do we ever want to print bytes from Python? (not a rhetorical question, I'm genuinely not sure)

Reply:
I don't think it makes sense to print bytes from Python. Anything going to the console should be a human readable string, and that's exactly what `str` is for and what `bytes` is not for.

Reply:
(We do have to convert the `str` to `bytes` to send it over the FFI but the rust code on the other side will convert it back to a Rust `String` immediately)